src/tiled/createpolygonobjecttool.cpp

```diff
@@ -62,7 +62,19 @@ CreatePolygonObjectTool::CreatePolygonObjectTool(QObject *parent)
     QIcon icon(QLatin1String(":images/24x24/insert-polygon.png"));
     icon.addFile(QLatin1String(":images/48x48/insert-polygon.png"));
     setIcon(icon);
-
+<<<<<<< HEAD
+<<<<<<< HEAD
+<<<<<<< HEAD
+
+=======
+    mtoolInfo = tr("<b>First Click</b> determines the location of object and first point of polygon. <b>Right Click</b> or press <b>Enter</b> to finish creating polygon. press <b>Escape</b> to cancel creation of polygon.");
+>>>>>>> 7b9ddd1b... added information to be displayed in status bar for object-tools, except not for template object tool
+=======
+    mtoolInfo = tr("<b>First Click</b> determines the location of object and first point of polygon. <b>Right Click</b> or press <b>Enter</b> to finish creating polygon. press <b>Escape</b> to cancel creation of polygon.");
+>>>>>>> 7b9ddd1b... added information to be displayed in status bar for object-tools, except not for template object tool
+=======
+    mtoolInfo = tr("<b>First Click</b> determines the location of object and first point of polygon. <b>Right Click</b> or press <b>Enter</b> to finish creating polygon. press <b>Escape</b> to cancel creation of polygon.");
+>>>>>>> 7b9ddd1b... added information to be displayed in status bar for object-tools, except not for template object tool
```

Comment:
These are conflicts markers, indicating to missed to resolve the conflict before committing (or before continuing the rebase). You can resolve these conflicts by either editing the file in your regular editor, or by using a special merge tool. I personally use [Meld](http://meldmerge.org/) with `git mergetool` for comfortably resolving conflicts.